pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "monocle_apptrace"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
should we just do 0.2.1 ??

Reply:
The versioning guide states that that for X.Y.Z , Z should be incremented if a patch is released with some fixes.
AN example is the opentelemetry SDK which mostly releases by incrementing Y if a new feature is releases and only updates Z when a patch(fix) is released